docs/UserGuide.md

```diff
@@ -227,9 +287,9 @@ Format: `findMedCon KEYWORD [MORE_KEYWORDS]`
 > * Patients matching at least one keyword will be returned (i.e. `OR` search).
 >    e.g. `diabetes` will return `Alex Yeoh`, `David Li`
 > * Returns an empty patient list panel if no matching patients with the given keywords are found.
->
-> Examples:
-> * `findMedCon diabetes arthritis` returns `Alex Yeoh` and `David Li`
+
+Example:
+ * `findMedCon diabetes arthritis` returns `Alex Yeoh` and `David Li`
```

Comment:
thinking about it, find appt would make sense but I'll just say it's satisfied by the panel on the right showing all the appointment timings